x/token/validation.go

```diff
@@ -97,3 +98,13 @@ func validateChange(change Attribute) error {
 func ValidateContractID(id string) error {
 	return class.ValidateID(id)
 }
+
+func canonicalKey(key string) string {
+	convert := map[string]string{
+		AttributeKeyImageURI.String(): AttributeKeyURI.String(),
+	}
```

Comment:
AttributeKeyImageURI and AttributeKeyURI are all fixed values, so convert also does not change. It is unnecessary to declare this every time. How about pre-declaring it with the var at the top?

Reply:
We should avoid using globals if it's not performance-critical. Also, there is no other place using this map.
Could you elaborate your concern on it?